docs/py_docs.rst

```diff
@@ -528,7 +570,7 @@ E.g., whilst documenting a ``lsst.afw.tables`` module, refer to a class in ``lss
 
 .. code-block:: rst
 
-   afw.detection.Footprint : Regular detection footprint.
+   `afw.detection.Footprint` : Regular detection footprint.
```

Comment:
a few lines below this one it looks like the back ticks are missing from `astropy.table.Tables`.